### PR TITLE
feat(pack): Added Azure Key Vault quickstart

### PR DESCRIPTION
**Pull request by EmilHammarstrand**
# Summary

Added new Azure Key Vault quickstart.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [x] Did you check that your alerts actually work?
- [x] Did you include an InstallPlan and Documentation reference?
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots
azure-key-vault

### CODE REVIEW

**Comment by josephgregoryii** [comment]
Hey @EmilHammarstrand ! Just wanted to let you know that we changed how the structure of quickstarts is represented in the repository earlier this week. I've gone ahead and updated this PR to represent those changes. 

TL;DR: Dashboards and alerts for a quickstart are moved into their own top level directory. You can read more about it in our [CONTRIBUTING.md](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md)

😄

**Review by josephgregoryii — APPROVED** [review]
